### PR TITLE
Created OrderService and tests for him

### PR DESCRIPTION
**Pull request by Nikita-Exeption**

### CODE REVIEW

File: src/main/java/org/nikita/chiken_bell/core/service/impl/OrderServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    public Optional<Order> getById(String id) {
+        return orders.stream().filter(x -> x.getId().equals(id)).findFirst();
+    }
+
+    @Override
+    public Collection<Order> getAll() {
+        return orders;
```
Comment (zxRasp) [inline src/main/java/org/nikita/chiken_bell/core/service/impl/OrderServiceImpl.java]: Should return unmodifiable copy of collection.

File: src/test/java/org/nikita/chiken_bell/core/service/impl/OrderServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+        assertEquals(2, service.getAll().size());
+        assertEquals(newOrder.getProducts(), CART.getProducts());
+    }
+
+    @Test
+    void getByIdPositiveCase() {
+        assertNotNull(service.getById(order.getId()));
+
+        Order getOrder = service.getById(order.getId()).orElseThrow(OrderNotFoundException::new);
```
Comment (zxRasp) [inline src/test/java/org/nikita/chiken_bell/core/service/impl/OrderServiceImplTest.java]: Shouldn't throw exception in test! Please use assertTrue(getOrder.isPresent())

**Review by zxRasp — CHANGES_REQUESTED** [review]
Please fix issues

File: src/test/java/org/nikita/chiken_bell/core/service/impl/OrderServiceImplTest.java (hunk 2 above)
Comment (Nikita-Exeption) [reply src/test/java/org/nikita/chiken_bell/core/service/impl/OrderServiceImplTest.java]: Ok

**Review by zxRasp — APPROVED** [review]

**Pull request merged**